src/index.ts

```diff
@@ -8,8 +9,9 @@ const program: Command = new Command();
 program
   .name("zetachain")
   .description("CLI tool for ZetaChain development.")
-  .version("1.0.0");
+  .version("dev");
 
-createCommand(program);
+program.addCommand(createCommand);
+program.addCommand(localnetCommand);
```

Comment:
Is it really this easy? Nice!

Reply:
Yeah. For toolkit, I think we'll either mount namespaces like `evm` and `solana` or import individual commands and group them into namespaces in the CLI repo. Doesn't really make a difference to end users, so not sure which path is better.